BraintreeCore/src/main/java/com/braintreepayments/api/AuthorizationProvider.java

```diff
@@ -6,7 +6,7 @@
  * Implement this interface to provide an asynchronous way for {@link BraintreeClient} to fetch
  * a client token from your server when it is needed.
  */
-interface AuthorizationProvider {
+public interface AuthorizationProvider {
```

Comment:
We are calling this AuthorizationProvider but the method name is `getClientToken` - should that be `getAuthorization`?

Reply:
Yeah that's a weird part since the `Authorization` type is no longer merchant-facing. Initially `ClientTokenProvider` / `getClientToken` went well together, but technically a merchant could provide a tokenization key here.

Reply:
This naming was also brought up in the DX GitHub [discussion](https://github.com/braintree/braintree_android/discussions/496#discussioncomment-2062004).